_component_design/selection-bucket.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Selection Bucket
+parent: Tables
+layout: component
+category: Components
+usage: >
+  A selection bucket contains lists of table values selected by the user. Each time a user selects a table value, a pill representing the selected value appears in the selection bucket. The selection bucket is located below the table and holds all selected values, across all table pages. The user can remove selected values from the selection bucket.
```

Comment:
A selection bucket is located...and contains selected values across all table pages.

Reply:
Done.